PayPal_Braintree/styles/modules/_apple-pay.scss

```diff
@@ -0,0 +1,17 @@
+.braintree-apple-pay-button {
+    -webkit-appearance: -apple-pay-button;
+    display: block;
+    padding: $spacer 0;
+    margin: $spacer--medium 0;
+    width: 200px;
+    min-height: 40px;
```

Comment:
shouldn't be 48px for the button as a minimal height?

Reply:
hmm it can be 48px :)